catalog/core/catalog-core-standardframework/src/test/java/ddf/catalog/impl/operations/ResourceOperationsOptionsTest.java

```diff
@@ -0,0 +1,323 @@
+
+import org.junit.Before;
+import org.junit.Test;
+
+import ddf.catalog.cache.solr.impl.ValidationQueryFactory;
+import ddf.catalog.content.impl.MockMemoryStorageProvider;
+import ddf.catalog.data.Result;
+import ddf.catalog.data.impl.MetacardImpl;
+import ddf.catalog.federation.FederationException;
+import ddf.catalog.federation.FederationStrategy;
+import ddf.catalog.filter.proxy.adapter.GeotoolsFilterAdapterImpl;
+import ddf.catalog.filter.proxy.builder.GeotoolsFilterBuilder;
+import ddf.catalog.impl.FrameworkProperties;
+import ddf.catalog.impl.QueryResponsePostProcessor;
+import ddf.catalog.operation.QueryRequest;
+import ddf.catalog.operation.QueryResponse;
+import ddf.catalog.plugin.PostIngestPlugin;
+import ddf.catalog.plugin.PostResourcePlugin;
+import ddf.catalog.resource.ResourceNotFoundException;
+import ddf.catalog.source.CatalogProvider;
+import ddf.catalog.source.UnsupportedQueryException;
+import ddf.catalog.util.impl.SourcePoller;
+import ddf.mime.MimeTypeResolver;
+import ddf.mime.MimeTypeToTransformerMapper;
+import ddf.mime.mapper.MimeTypeMapperImpl;
+
+public class ResourceOperationsOptionsTest {
+    ResourceOperations resourceOperations;
+
+    FrameworkProperties frameworkProperties;
+
+    OperationsSecuritySupport opsSecuritySupport;
+
+    SourcePoller mockPoller;
+
+    CatalogProvider provider;
+
+    FederationStrategy mockFederationStrategy;
+
+    ArrayList<PostIngestPlugin> postIngestPlugins;
+
+    MockMemoryStorageProvider storageProvider;
+
+    MimeTypeResolver mimeTypeResolver;
+
+    MimeTypeToTransformerMapper mimeTypeToTransformerMapper;
+
+    List<PostResourcePlugin> mockPostResourcePlugins;
+
+    QueryResponse queryResponseMock;
+
+    QueryOperations queryOperationsMock;
+
+    boolean fanoutEnabled;
+
+    MetacardImpl metacard;
+
+    ArrayList<Result> mockResultList;
+
+    Result resultMock;
+
+    @Before
+    public void setUp() throws Exception {
+        setUpFrameworkProperties();
+        setUpResourceOperationsAndMocks();
+    }
+
+    private void setUpResourceOperationsAndMocks() throws URISyntaxException {
+        queryOperationsMock = mock(QueryOperations.class);
+        queryResponseMock = mock(QueryResponse.class);
+        resultMock = mock(Result.class);
+
+        metacard = new MetacardImpl();
+        metacard.setId("Bobbert");
+        metacard.setTitle("Bobbert's Title");
+
+        mockResultList = new ArrayList<>();
+
+        resourceOperations = new ResourceOperations(frameworkProperties,
+                queryOperationsMock,
+                opsSecuritySupport);
+    }
+
+    private void setUpFrameworkProperties() {
+        frameworkProperties = new FrameworkProperties();
+        frameworkProperties.setAccessPlugins(new ArrayList<>());
+        frameworkProperties.setPolicyPlugins(new ArrayList<>());
+        frameworkProperties.setSourcePoller(mockPoller);
+        frameworkProperties.setCatalogProviders(Collections.singletonList((CatalogProvider) provider));
```

Comment:
Not critical, simply included as a part of setting different framework properties. Removed as it does not break the tests in this case.